srtcore/api.h

```diff
@@ -267,30 +267,32 @@ friend class CRendezvousQueue;
 // Debug support
 inline std::string SockaddrToString(const sockaddr* sadr)
 {
-	void* addr =
+    void* addr =
         sadr->sa_family == AF_INET ?
-            (void*)&((sockaddr_in*)sadr)->sin_addr
+        (void*)&((sockaddr_in*)sadr)->sin_addr
         : sadr->sa_family == AF_INET6 ?
-            (void*)&((sockaddr_in6*)sadr)->sin6_addr
+        (void*)&((sockaddr_in6*)sadr)->sin6_addr
         : 0;
-	// (cast to (void*) is required because otherwise the 2-3 arguments
-	// of ?: operator would have different types, which isn't allowed in C++.
+    // (cast to (void*) is required because otherwise the 2-3 arguments
+    // of ?: operator would have different types, which isn't allowed in C++.
     if ( !addr )
         return "unknown:0";
 
-	std::ostringstream output;
-	char hostbuf[1024];
-	if (!getnameinfo(sadr, sizeof(*sadr), hostbuf, 1024, NULL, 0, NI_NAMEREQD))
-	{
-		output << hostbuf;
-	}
-	else
-	{
-		output << "unknown";
-	}
-
-	output << ":" << ntohs(((sockaddr_in*)sadr)->sin_port); // TRICK: sin_port and sin6_port have the same offset and size
-	return output.str();
+    std::ostringstream output;
+    char hostbuf[1024];
+    if (!getnameinfo(sadr, sizeof(*sadr), hostbuf, 1024, NULL, 0, NI_NAMEREQD))
+    {
+        output << hostbuf;
+    }
+    else
+    {
+        strcpy(hostbuf, "unknown");
+        inet_ntop(sadr->sa_family, addr, hostbuf, 1023);
```

Comment:
Man page says "The caller specifies the number of bytes available in this buffer in the argument size", why 1023 and not sizeof(hostbuf)?